docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -285,14 +285,13 @@ template: signature_info_template
 subject: |
   I plan to **e-file**. What should I choose?
 content: |
-  If you **plan to e-file**, you may want to add your electronic signature. Otherwise you will need to print your forms, sign them by hand, and then scan them.
-
-  To find out if your court has e-filing, read the [What is E-Filing?](https://michiganlegalhelp.org/resources/mifile/what-e-filing) article.
-  The article tells you how to find out which courts have e-filing for which case types.
+  If you **plan to e-file**, you may want to sign electronically. Choose *Yes* to add your electronic signature now.
   
-  ${ MLH_case_type_language }
```

Comment:
@normon66 I think we covered this in the meeting but just in case--I had just moved the case type language, not deleted it.  But now that I have a better understanding, I moved it back so it will stay its own paragraph.